dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp

```diff
@@ -34,6 +34,7 @@
                 value="strInode"
             />
             <div class="sideMenuWrapper">
+                <div dojoAttachPoint="structures_select"></div>

```

Comment:
not sure if this one could be change to content_type_select since structures is a legacy term that we should avoid, otherwise would be ok